### DISCUSSION TITLE
Picklist missing

### DISCUSSION THREAD

**Discussion by chrisdevelop777555** [YetiForceCompany/YetiForceCRM#13999] [General]
When going through a test sale a picklist comes up with for example Hardware, Software etc dispite going to ever selection and checking each picklist I cannot find this picklist selection to delete or add to.  Does anyone have any idea where i can find this or is this a bug thats failing to display it

**Comment by rskrzypczak** [comment]
I think you mean folders (tree elements) not a picklist.
![image](https://user-images.githubusercontent.com/10221999/103612120-04a0dc00-4f24-11eb-90f1-f21bc1ce2f8b.png)
